git-and-github/code-versioning.md

```diff
@@ -0,0 +1,59 @@
+# Git Tagging & Releases
+
+## What is Git Tagging?
+
+  * [Tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) is a simple, easy way to keep track of your code at a certain point in time. Some instances where this becomes useful are:
+    * Client-facing deliverables such as milestones, UAT, or production launch/release
+    * Internal-facing code at a point that you or your team wants to demarcate as important
+    * Versioning of your code (such as following SemVer or other release formats)
+  * Because we are not following a strict rule or methodology, each of our teams can decide for themselves which method of tag naming they want to follow. You could go strict and version your code based on milestones or releases, or you could use human-readable tag names. In the end, the power is _yours_!
```

Comment:
I would challenge us to find and commit to one way to do this so we aren't reinventing the wheel here with each project.

Reply:
That being said, as a whole I like this version, and think it's a good place to start.

Reply:
@ryekerjh `major.minor.patch`